CONTRIBUTING.md

```diff
@@ -248,100 +136,44 @@ In addition, the following conventions are used in Acts code:
-git reset --hard <hash of B>
-git checkout <new_branch_name>
-```
-
-which should give the following situation:
-<img src="doc/figures/move_to_branch2.png" alt="moving commits to new branch">
-Now, master is pointing to B, HEAD and &lt;new\_branch\_name&gt; are pointing to E and you can happily continue with your work.
+### <a name="tips-users-gitlab">Tips for users migrating from Gitlab</a>
+- The most obvious difference first: What is called Merge Request (MR) in Gitlab is called Pull Request (PR) in Github.
+- Once your PR is ready to be merged, request a review by the users in the [reviewers team](https://github.com/orgs/acts-project/teams/reviewers)
+- The access rights model of Github is slightly different from Gitlab. As Acts started enforcing using your own fork with the switch to Github, developers no longer have write access to the upstream repository. Unfortunately, approval of a PR by users without write access does not automatically liberate the merge button for users with write access. Instead, users with write access have to manually check if a review from an authorized person is present before merging.
 
 ## <a name="review-other-contributions">Review other contributions</a>
 
-### <a name="approving-a-merge-request">Approving a merge request</a>
+Acts requires that every pull request receives at least one review by a member of the reviewers team before being merged but anyone is welcome to contribute by commenting on code changes. You can help reviewing proposed contributions by going to [the "pull requests" section of the Acts (core) Github repository](https://github.com/acts-project/acts-core/pulls).
 
-Writing new code is not the only way one can contribute to the Acts project. Another greatly appreciated contribution is looking at other proposed contributions to the project. The more eyeballs look at a piece of code before it is merged into the Acts repository, the smaller the chances that a bug or other unwanted behaviour is accidentally introduced into the Acts codebase. This is why we require that every Acts merge request receives at least one human review before being merged.
+As some of the guidelines recommended here require rights granted to the reviewers team, this guide specifically addresses the people in this team. The present contribution guide should serve as a good indication of what we expect from code submissions.
 
-You can help reviewing proposed contributions by going to [the "merge requests" section of the Acts Gitlab repository](https://gitlab.cern.ch/acts/acts-core/merge_requests) and having a look at the proposals that are being made here. The present contribution guide should serve as a good indication of what we expect from code submissions. In addition, please look at the merge request itself:
+### <a name="apply-labels">Configuring your own PRs and PRs by people with read rights</a>
+
+* Check if the "request review" label is set (equivalent to WIP tag in title of a MR in Gitlab)
```

Comment:
This is now actually possible in Github, with me adding the WIP bot. So either use Draft PRs, oder add the "WIP" prefix to the PR title, and this will block the PR from being merged.

Reply:
adapted accordingly